recipe/meta.yaml

```diff
@@ -45,6 +45,11 @@ requirements:
     - numbagg
     - flox
     - opt_einsum
+    - qt-niu
+    - pyvideoreader
+    - napari
```

Comment:
similarly, we restrict napari to >0.5.

I general, do we apply the same pins for PyPI and conda?

Reply:
Not sure, but it sounds like a good idea?

Reply:
I'll test this by pushing here.

Reply:
nvm, you beat me to it.